### ISSUE TITLE
[Question] Typescript on Client

### ISSUE THREAD

**Issue by axemclion** [asrytis/dice-game#1]
I see that while the server is written in TypeScript, the ReactNative client is still JavaScript. Am I missing something, or is the client indeed in ES6 ?

**Comment by asrytis** [comment]
You are totally right. Only the server is done in TypeScript. I was considering using Flow for client but it didn't happen.

**Comment by axemclion** [comment]
@asrytis I was thinking you could use Typescript for the client also. You could use the `--transformer` flag and just pick up the `.tsx` files using the Typescript compiler. Do you think that would work for you ?

**Comment by asrytis** [comment]
@axemclion yes I may still switch to TypeScript if I have time. Type safety is not a huge concern in this particular client, but I'd love to see how code completion supplements an already fantastic dev experience.
